module/VuFind/src/VuFind/Db/Service/UserServiceFactory.php

```diff
@@ -69,8 +69,11 @@ public function __invoke(
         }
         $config = $container->get(\VuFind\Config\PluginManager::class)
             ->get('config');
+        $userTable = $container->get(\VuFind\Db\Table\PluginManager::class)
+          ->get('user');
         $userService = parent::__invoke($container, $requestedName, $options);
         $userService->setConfig($config);
+        $userService->setUserTable($userTable);
```

Comment:
As above, it might make sense for this to be a mandatory constructor dependency. Config was injected optionally since the code still works with the null default.